src/DataViews/SamplingReportDataView.cpp

```diff
@@ -53,8 +53,8 @@ const std::vector<DataView::Column>& SamplingReportDataView::GetColumns() {
     columns.resize(kNumColumns);
     columns[kColumnSelected] = {"Hooked", .0f, SortingOrder::kDescending};
     columns[kColumnFunctionName] = {"Name", .4f, SortingOrder::kAscending};
-    columns[kColumnInclusive] = {"Inclusive", .0f, SortingOrder::kDescending};
-    columns[kColumnExclusive] = {"Exclusive", .0f, SortingOrder::kDescending};
+    columns[kColumnInclusive] = {"Inclusive, %", .0f, SortingOrder::kDescending};
```

Comment:
I am not sure how we do it in the other tabs. But maybe "Inclusive (%)"? But I let this up to you!

Reply:
I've not found any other tabs' names including the units.